src/users/users.service.ts

```diff
@@ -15,4 +16,15 @@ export class UsersService {
     const { password, ...result } = user;
     return result;
   }
+
+  async getUserByUserId(userDto: UserDto) {
+    const { userId } = userDto;
+    const user = await this.prisma.user.findUnique({
+      where: {
+        id: userId,
+      },
+    });
+
+    return user;
```

Comment:
사용자 상세 조회 API에서 비밀번호도 함께 보여져서 요기서도 비밀번호는 제외해도 좋을 것 같아요!
근데 password 필드를 공통적으로 제외할 더 좋은 방법이 있을 것 같긴한데 잘 모르겠네요..

```suggestion
    const { password, ...result } = user;
    return user;
```